griddb/src/main/java/com/yahoo/ycsb/db/package-info.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) 2018 TOSHIBA Digital Solutions Corporation.
+ * Copyright (c) 2018 YCSB contributors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you
+ * may not use this file except in compliance with the License. You
+ * may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ * implied. See the License for the specific language governing
+ * permissions and limitations under the License. See accompanying
+ * LICENSE file.
+ */
+
+/**
+ * The YCSB binding for <a href="https://griddb.net/">GridDB</a>.
+ */
+package com.yahoo.ycsb.db;
```

Comment:
all the binding classes should be in a griddb specific package

Reply:
I changed package name by 10c6705abff4632bee8e6781f0e6662179e78b55 .